velox/expression/fuzzer/SparkSpecialFormSignatureGenerator.cpp

```diff
@@ -22,15 +22,28 @@ SparkSpecialFormSignatureGenerator::getSignaturesForCast() const {
   std::vector<exec::FunctionSignaturePtr> signatures =
```

Comment:
Apology for the misrepresentation. It seems more appropriate for me to focus on moving the API in this PR and leave the fuzzer test change for another, and I only meant to cover the fix for empty exponent. Thank you!